charts/plane-ce/templates/workloads/worker.deployment.yaml

```
@@ -45,4 +45,44 @@ spec:
+  labels:
+    app.name: {{ .Release.Namespace }}-{{ .Release.Name }}-worker-hpa
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: {{ .Release.Name }}-worker-wl
+  minReplicas: {{ .Values.autoscaling.minReplicas }}
+  maxReplicas: {{ .Values.autoscaling.maxReplicas }}
+  metrics:
+    {{- if .Values.autoscaling.targetCPUUtilizationPercentage }}
+    - type: Resource
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)